README.md

```diff
@@ -59,7 +59,7 @@ The following additional configuration options are available by specifying them
 
 We will always endeavor to support the latest stable version of TypeScript.
 
-The version of TypeScript currently supported by this parser is `~3.0.1`. This is reflected in the `devDependency` requirement within the package.json file, and it is what the tests will be run against. We have an open `peerDependency` requirement in order to allow for experimentation on newer/beta versions of TypeScript.
+The version of TypeScript currently supported by this parser is `~3.1.1`. This is reflected in the `devDependency` requirement within the package.json file, and it is what the tests will be run against. We have an open `peerDependency` requirement in order to allow for experimentation on newer/beta versions of TypeScript.
```

Comment:
this line is just chore, parser right now uses 3.1.1

Reply:
Correct - this actually hasn't changed. Just updating it to be accurate!

Reply:
However, semver applies to the specification rather than implementation. This description looks the spec.

> The version of TypeScript currently supported by this parser is `~3.1.1`.

Reply:
@mysticatea I'm not sure I follow. Do you mind clarifying what you think this should say instead?

Reply:
Ah, I got it. I confirmed that the documentation is not updated. I'm sorry for the confusion.

Reply:
No problem! Thanks for double checking.